include/tvm/tir/ir_builder.h

```diff
@@ -29,6 +29,7 @@ namespace tir {
 
 using tvm::runtime::NDArray;
 using tvm::tir::Buffer;
+using tvm::tir::IterVar;
```

Comment:
remove this - not used

Reply:
move it to `ir_builder.cc`?
it is used there

Reply:
got it

Reply:
go ahead